targets/e17

```diff
@@ -11,7 +11,10 @@ HOSTBIN='starte17'
 . "${TARGETSDIR:="$PWD"}/common"
 
 ### Append to prepare.sh:
-install e17
+if [ "${DISTROAKA:-"$DISTRO"}" = 'gentoo' ]; then
+    set_flags xrandr xcomposite
```

Comment:
Yes, these flags aren't used in other packages, the functionality is forced on. Gentoo is weird sometimes. :)